### DISCUSSION TITLE
[blocks]: Interactive World Map Graph Component

### DISCUSSION THREAD

**Discussion by yannik2480** [shadcn-ui/ui#6011] [Blocks Request]
## Description  
I would love to see a world map graph component integrated into Shadcn, similar to the [amCharts Multi-Series Map](https://www.amcharts.com/demos/multi-series-map/). This component would enable users to display interactive geographical data with multi-series support and customizable markers.

## Use Cases  
- Global sales, population, or trend visualization.  
- Visualizing supply chains and logistics networks.  
- Educational geography and data analysis tools.  

## Implementation Suggestions  
- Use libraries like [Mapbox GL JS](https://docs.mapbox.com/mapbox-gl-js/) or [D3.js](https://d3js.org/) for map rendering and interactivity.  
- Align the component with Shadcn’s composable and customizable design philosophy.  

### Example

https://www.amcharts.com/demos/multi-series-map/

**Comment by thiagovilla** [comment]
Second that!
